server/server.go

```diff
@@ -81,14 +80,20 @@ func NewServer(globalConfiguration GlobalConfiguration) *Server {
 	currentConfigurations := make(configs)
 	server.currentConfigurations.Set(currentConfigurations)
 	server.globalConfiguration = globalConfiguration
-	server.loggerMiddleware = middlewares.NewLogger(globalConfiguration.AccessLogsFile)
-	server.accessLoggerMiddleware = accesslog.NewLogHandler()
 	server.routinesPool = safe.NewPool(context.Background())
 	if globalConfiguration.Cluster != nil {
 		// leadership creation if cluster mode
 		server.leadership = cluster.NewLeadership(server.routinesPool.Ctx(), globalConfiguration.Cluster)
 	}
 
+	if server.globalConfiguration.AccessLogsFile != "" {
+		var err error
+		server.accessLogFile, err = os.OpenFile(server.globalConfiguration.AccessLogsFile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+		if err != nil {
+			log.Error("Error opening file", err)
+		}
+	}
```

Comment:
Is this added block strictly related to the middleware change this PR is primarily about?